building-web-forms/submitting-user-data/README.md

```diff
@@ -0,0 +1,100 @@
+# Text
+
+In this lesson, we will learn about what happens when a user submits a form. We will also learn about where the data goes, and how we can check, what data is sent to the web server.
+
+First, we'll discuss what happens to the data when we submit a form. To understand that, we need to have an understanding of Client/Server architecture.
+
+## Client/Server architecture
+
+The Web uses a Client/Server architecture that can be summarized as follows:
+
+1. Client (a web browser) sends a request,
+2. To a Server (a web server like Apache, Tomcat, etc.),
+3. Using the [HTTP protocol](https://developer.mozilla.org/en-US/docs/Web/HTTP).
+
+The server answers the request using the same protocol.
+
+The HTML form we create is a user-friendly way to configure an HTTP request to send data to a server. This enables the user to provide information to be delivered in the HTTP request.
+
+## Sending data
+
+The `form` element defines how the data will be sent to the web server. All of its attributes are designed to let you configure the request to be sent when a user clicks a submit button.
+
+The two most important attributes of a form element are `action` and `method`.
+
+### `action` attribute
+
+The `action` attribute defines where the data gets sent. Its value must be a valid URL. If this attribute isn't provided, the data will be sent to the URL of the page containing the form — the current page.
+
+```js
+
+   <form action="https://google.com"> #External URL
+   <form action="/localserver"> #Local URL
+
+```
+
+The names and values of the form controls are sent to the server as `name=value` pairs joined with the `&` symbol.
```

Comment:
An example for this statement would be helpful.

Reply:
The examples are stated below in the method related explanations.